autoware_launch/launch/autoware.launch.xml

```diff
@@ -119,9 +119,10 @@
 
   <!-- Control -->
   <group if="$(var launch_control)">
-    <include file="$(find-pkg-share control_launch)/launch/control.launch.xml">
-      <!-- options for lateral_controller_mode: mpc_follower, pure_pursuit -->
-      <arg name="lateral_controller_mode" value="mpc_follower"/>
+    <include file="$(find-pkg-share control_launch)/launch/control.launch.py">
```

Comment:
We should use control.launch.py since control.launch.xml is just a wrapper for control.launch.py.

FYI: control.launch.xml was removed in tier4_control_launch.
https://github.com/autowarefoundation/autoware.universe/tree/main/launch/tier4_control_launch/launch